astropy/modeling/functional_models.py

```diff
@@ -440,6 +439,92 @@ def inverse(self):
         return inv
 
 
+class Sersic1D(Fittable1DModel):
+    r"""
+    One dimensional Sersic surface brightness profile.
+
+    Parameters
+    ----------
+    amplitude : float
+        Central surface brightness, within r_eff. 
+    r_eff : float
+        Effective (half-light) radius
+    n : float
+        Sersic Index.
+
+    See Also
+    --------
+    Gaussian1D, Moffat1D, Lorentz1D
+
+    Notes
+    -----
+    Model formula:
+
+    .. math::
+
+        I(r)=I_e exp\left[-b_n\left(\frac{r}{r_{e}}\right)^{(1/n)}-1\right]
+
+    The constant :math:`b_n` is defined such that :math:`r_e` contains half the total 
+    luminosity, and can be solved for numerically.  
+
+    .. math:: 
+
+        \Gamma(2n) = 2\gamma (b_n,2n)
+
+    Examples
+    --------
+    .. plot::
+        :include-source:
+
+        import numpy as np
+        from astropy.modeling.models import Sersic1D
+        import matplotlib.pyplot as plt
+
+        plt.figure()
+        plt.subplot(111,xscale='log',yscale='log')
+        s1 = Sersic1D(amplitude=1, r_eff=5)
+        r=np.arange(0,100,.01)
+
+        for n in range(1,10):
+             s1.n = n 
+             plt.plot(r,s1(r),color='k',alpha=0.5,lw=2)
+
+        plt.axis([1e-1,30,1e-2,1e3])
+        plt.xlabel('log Radius',fontsize=16)
+        plt.ylabel('log Surface Brightness',fontsize=16)
+        plt.text(.25,1.5,'n=1',fontsize=16)
+        plt.text(.25,300,'n=10',fontsize=16)
+        plt.xticks([])
+        plt.yticks([])
+        plt.show()
```

Comment:
(Added #3894 for anyone who would like to take that on--it would also make for a good task for a sprint)